src/shapes/text.class.js

```diff
@@ -913,6 +924,9 @@
      * @param {String} [prevGrapheme] character preceding the one to be measured
      */
     _getGraphemeBox: function(grapheme, lineIndex, charIndex, prevGrapheme, skipLeft) {
+      if (grapheme == null) {
+        grapheme = this._textLines[lineIndex][charIndex];
+      }
```

Comment:
fabric is approaching 30k lines of code. i think it reached 300k of minified size. If not something that the user can break or that some browser needs in particular, i do not put it inside.